app/ui/src/app/integration/import-export/integration-import-export.component.html

```diff
@@ -0,0 +1,7 @@
+<ul class="row nav nav-tabs nav-tabs-pf toolbar-pf">
+  <li routerLinkActive="active">
+    <a [routerLink]="['api-connector']">Import Integration</a>
```

Comment:
'api-connector' seems like an odd link for import integration.

Reply:
This is copy-pasta.